addons/google_calendar/utils/google_calendar.py

```diff
@@ -70,7 +70,10 @@ def insert(self, values, token=None, timeout=TIMEOUT):
     def patch(self, event_id, values, token=None, timeout=TIMEOUT):
         url = "/calendar/v3/calendars/primary/events/%s?sendUpdates=all" % event_id
         headers = {'Content-type': 'application/json', 'Authorization': 'Bearer %s' % token}
-        self.google_service._do_request(url, json.dumps(values), headers, method='PATCH', timeout=timeout)
+        try:
+            self.google_service._do_request(url, json.dumps(values), headers, method='PATCH', timeout=timeout)
+        except requests.HTTPError as e:
+            raise e
```

Comment:
Putting everything in try except will have performance impact. Did you tried to measure them?
We are not found of putting everything in try except blocks because it makes the code comprehension more difficult. before, you had one try except and now you have three.